worlds/stardew_valley/data/region_data.py

```diff
@@ -119,3 +119,35 @@ class SVRegion:
     mines_floor_115 = "The Mines - Floor 115"
     mines_floor_120 = "The Mines - Floor 120"
 
+
+class DeepWoodsRegion:
+    entrance_to_deep_woods = "Entrance to the Deep Woods"
```

Comment:
The entrance to the Deep Woods is technically "Floor 1"; however its a special map which contains an abandoned house, a sign, five mysterious pillars, and a path to the left and right which lead into the "actual" Deep Woods.  When you use the obscured path in Secret Woods to reach the Deep Woods, this is the area you reach, which is really an "entrance" more than anything; it would be like naming the entrance of the mines "Mine Level 0", no?

Reply:
Yeah that's fair then. That's how we named the other similar regions as well